modules/lib/lib-node/src/test/resources/test/FindNodesByQueryHandlerTest.js

```diff
@@ -55,6 +55,21 @@ var expectedEmptyJson = {
     'hits': []
 };
 
+var expectedSortJson = {
+    "total": 12902,
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8807034)
```suggestion
    'total': 12902,
```